README.md

```diff
@@ -74,6 +74,8 @@ for administration of the Hub and its users.
     The `nodejs-legacy` package installs the `node` executable and is currently
     required for npm to work on Debian/Ubuntu.
 
+- A [pluggable authentication module (PAM)](https://en.wikipedia.org/wiki/Pluggable_authentication_module) 
+  to use the default Authenticator
```

Comment:
Can you add the condition: "if using the default PAM Authenticator"?

Reply:
Sure, `Readme` just modified.